src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. The line 2 should be removed.